edx_proctoring/templates/proctoring/seq_proctored_exam_submitted.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have submitted this proctored exam for review
+    {% endblocktrans %}
+  </h3>
+
```

Comment:
@chrisndodge How about a text paragraph here, under the h3 and before the h4?
What do you think of this:
<p>Make sure you return to the proctoring software and select **Quit** to end the proctoring session.</p>

..and I am also going to modify the text below the h4.